RedditComponent/graph.py

```diff
@@ -45,26 +45,26 @@ def uploadImage():
 	submitReddit(theUrl)
 	
 def drawGraph(x, y1, y2):
-    x = pd.to_datetime(pd.Series(x)).dt.strftime('%b-%d') # Clean Up X Labels
-    
-    fig, axs = plt.subplots(facecolor='w', nrows=2) # Set Up Figure and Subplots
-    
-    sns.barplot(x=x, y=y1, color=sns.color_palette()[0], ax=axs[0]) # Plot Barplots
-    sns.barplot(x=x, y=y2, color=sns.color_palette()[0], ax=axs[1])
-    
-    axs[0].set_ylabel('Yes Votes') # Clean Y Labels
-    axs[1].set_ylabel('No Votes')
-    
-    max_y = max(max(y1),max(y2)) # Synchronize Axes
-    axs[0].set_ylim(0,max_y)
-    axs[1].set_ylim(0,max_y)
-    
-    plt.tight_layout() # Clean Suptitle
-    fig.suptitle('Weekly Voting Data', y=1.03)
+    	x = pd.to_datetime(pd.Series(x)).dt.strftime('%b-%d') # Clean Up X Labels
+    	
+    	fig, axs = plt.subplots(facecolor='w', nrows=2) # Set Up Figure and Subplots
+    	
+    	sns.barplot(x=x, y=y1, color=sns.color_palette()[0], ax=axs[0]) # Plot Barplots
+    	sns.barplot(x=x, y=y2, color=sns.color_palette()[0], ax=axs[1])
+    	
+    	axs[0].set_ylabel('Yes Votes') # Clean Y Labels
+    	axs[1].set_ylabel('No Votes')
+    	
+    	max_y = max(max(y1),max(y2)) # Synchronize Axes
+    	axs[0].set_ylim(0,max_y)
+    	axs[1].set_ylim(0,max_y)
+    	
+    	plt.tight_layout() # Clean Suptitle
+    	fig.suptitle('Weekly Voting Data', y=1.03)
```

Comment:
change to "plt" instead of fig, remove y=1.03. This removes issue of title running into graph as well as the issue of title not appearing in image.